container/common/helpers.go

```diff
@@ -143,7 +152,7 @@ func readString(dirpath string, file string) string {
 
 func readUInt64(dirpath string, file string) uint64 {
 	out := readString(dirpath, file)
-	if out == "" {
+	if out == "" || out == "max" {
```

Comment:
why is this needed?

Reply:
ah nm, `max` is the value in `/sys/fs/cgroup/pids/.../pids.max` when the max is unbounded :+1:

Reply:
That's right.